src/logika/Grid.java

```diff
@@ -334,20 +351,14 @@ public static Grid readFromFile(String filename, int size) throws FileNotFoundEx
 		return grid;
     }
 	
-	public Grid placeColorAndCopy(Index idx, FieldColor color) {
+	public Grid deepcopy() {
 		Grid newGrid = new Grid(height, width);
```

Comment:
Tu raje uporabi konstruktor `Grid(height, width, grid)`, tako da se znebimo ene 8*8 for zanke.
PS. spremebmo je treba prekopirat še v mapo Tekmovanje

Reply:
Funkcija se pokliče enkrat na chooseMove, praktično nima vpliva na performance.
Nočem pisati čez originalni Grid, če sem slučajno kak bug izpustil v prepisovanju.